.eslintrc.js

```diff
@@ -8,9 +8,15 @@ module.exports = {
   },
 
   "env": {
+    "browser": true,
     "es6": true,
+    "mocha": true,
     "node": true,
-    "browser": true
+  },
+
+  "globals": {
+    "expect": true,
```

Comment:
this is not needed, I inject the  globals via webpack eslint loader - see test-web and test-node configs
setting this here will mean that eslint will think you're in mocha all the time - also in dev mode...